src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/GatherTranches.java

```diff
@@ -21,7 +23,8 @@
         oneLineSummary = "Gathers scattered VQSLOD tranches into a single file",
         programGroup = VariantProgramGroup.class
 )
-@DocumentedFeature
+@DocumentedFeature(enable=false)
```

Comment:
Forgot to comment on this: what's up with the `enable=false` here?

Reply:
This will cause the doc for the feature to not be included in the output. The enable property was ported over from the GTAK3 doc system, though I'm not sure what purpose it served there. Although it will work here (it will hide the doc), you can achieve the same effect achieved by just removing the `@DocumentedFeature` annotation altogether.